src/lib/expansion/expansion-panel.html

```diff
@@ -1,8 +1,12 @@
 <ng-content select="mat-expansion-panel-header"></ng-content>
-<div [class.mat-expanded]="expanded" class="mat-expansion-panel-content"
-     [@bodyExpansion]="_getExpandedState()" [id]="id">
+<div class="mat-expansion-panel-content"
+     [class.mat-expanded]="expanded"
+     [@bodyExpansion]="_getExpandedState()"
+     (@bodyExpansion.done)="_animationDone.next()"
+     [id]="id">
   <div class="mat-expansion-panel-body">
-    <ng-content></ng-content>
+    <ng-template><ng-content></ng-content></ng-template>
```

Comment:
This doesn't stop Angular from initializing stuff; we can make this change to stop stuff from making it into the DOM, but we'll ultimately need an alternate API where users provide an `ng-template`

Reply:
Ah, I was under the impression that this was the reason we do it the way we do for the tabs. In that case I'll rework it to use a `@ContentChild` to find an `ng-template` and instantiate it.

Reply:
@crisbeto - Look at what I did for https://github.com/angular/material2/pull/6832